### DISCUSSION TITLE
No Folder Structure in GitHub Desktop Repository list

### DISCUSSION THREAD

**Discussion by YoricW** [community/community#22982] [New to GitHub]
<p>Hello :slight_smile:</p>
<p>GitHub Desktop Repository list <strong>does not seem to have any folder structure</strong> feature</p>
<p>I can only add a repository<br>
not add a folder with multiple repositories inside</p>
<p>I have no way to orginize or SubGroup my repository list<br>
All 30+ repositories are just in 1 AlphaNumeric list<br>
with no Grouping or SubGrouping possibilities</p>
<p><strong>Am I missing something?</strong><br>
<strong>Or is it possible to Implement such a feature?</strong></p>
<p>GitLab has a feature where you can orginize projects(Repositories) into Groups<br>
Like a folder structure</p>
<p>Thank you for your time</p>

**Comment by steveward** [comment, marked as answer]
<p>Thanks for the feedback about GitHub Desktop <a class="mention" href="/yoricw">@yoricw</a>. </p>
<p>&gt;I can only add a repository<br>
not add a folder with multiple repositories inside</p>
<p>At this time there is no option to scan a folder to discover all of the repositories available, but this is something we are tracking in <a href="https://github.com/desktop/desktop/issues/1574" rel="nofollow noopener">https://github.com/desktop/desktop/issues/1574. </a>The best way to add multiple repositories is to select them all in File Explorer and then drag-and-drop them directly into GitHub Desktop.</p>
<p>&gt;I have no way to orginize or SubGroup my repository list<br>
All 30+ repositories are just in 1 AlphaNumeric list<br>
with no Grouping or SubGrouping possibilities</p>
<p>This is something that we discussed in <a href="https://github.com/desktop/desktop/issues/6460" rel="nofollow noopener">https://github.com/desktop/desktop/issues/6460</a>. At this time we do not have any further plans to make changes to how repositories are organized, but this may be something we revisit again in the future. You can find some further context in that issue.</p>
<p>I hope that is helpful. Please feel free to drop any further feedback you have into the issues I linked – we especially appreciate hearing feedback about how a certain feature would improve your workflow with GitHub Desktop.</p>
